lang/it.json

```diff
@@ -0,0 +1,1010 @@
+        "pdp-sale-price-label": "Ora:",
+        "pdp-non-sale-price-label": "Prima:",
+        "pdp-retail-price-label": "Prezzo di listino:",
+        "pdp-custom-fields-tab-label": "Informazioni aggiuntive"
+    },
+    "consent_manager": {
+        "data_collection_warning": "Utilizziamo i cookie (e tecnologie simili) per raccogliere dati con cui migliorare la tua esperienza di acquisto.",
+        "accept_all_cookies": "Accetta tutti i cookie",
+        "gdpr_settings": "Impostazioni",
+        "data_collection_preferences": "Preferenze per la raccolta dei dati sul sito web",
+        "use_data_by_cookies": "usa i dati raccolti dai cookie e dalle librerie JavaScript per migliorare la tua esperienza di acquisto.",
+        "data_categories_table": "La tabella di seguito illustra come vengono utilizzati questi dati per categoria. Per disattivare una categoria di raccolta dati, seleziona \"No\" e salva le tue preferenze.",
+        "allow": "Consenti",
+        "accept": "accetta",
+        "deny": "Nega",
+        "dismiss": "Ignora",
+        "reject_all": "Rifiuta tutto",
+        "category": "Categoria",
+        "purpose": "Scopo",
+        "functional_category": "Funzionali",
+        "functional_purpose": "Consente funzionalità avanzate, come video e chat dal vivo. Se non acconsenti, alcune o tutte queste funzioni potrebbero non funzionare correttamente.",
+        "analytics_category": "Analisi",
+        "analytics_purpose": "Fornire informazioni statistiche sull'utilizzo del sito, ad es. analisi web, per permetterci di migliorare questo sito.",
+        "targeting_category": "Targeting",
+        "advertising_category": "Pubblicità",
+        "advertising_purpose": "Utilizzato per creare profili o personalizzare i contenuti per migliorare la tua esperienza di acquisto.",
+        "essential_category": "Essenziali",
+        "esential_purpose": "Essenziali per il funzionamento del sito e dei servizi richiesti, ma non svolgono alcuna funzione secondaria o aggiuntiva.",
+        "yes": "Sì",
+        "no": "No",
+        "not_available": "N.D.",
+        "cancel": "Annulla",
+        "save": "Salva",
+        "back_to_preferences": "Torna alle preferenze",
+        "close_without_changes": "Sono state apportate modifiche non salvate alle preferenze di raccolta dati. Chiudere senza salvare?",
+        "unsaved_changes": "Sono state apportate modifiche non salvate",
+        "by_using": "Utilizzando il nostro sito, accetti i",
+        "agree_on_data_collection": "Utilizzando il nostro sito, acconsenti alla raccolta di dati come descritto nella nostra ",
+        "change_preferences": "Puoi modificare le tue preferenze in qualsiasi momento",
+        "cancel_dialog_title": "Vuoi davvero annullare?",
+        "privacy_policy": "Informativa sulla privacy",
+        "allow_category_tracking": "Consenti il tracciamento di [CATEGORY_NAME]",
+        "disallow_category_tracking": "Non consentire il tracciamento di [CATEGORY_NAME]"
+    }
+}
```

Comment:
@BC-tymurbiedukhin should we add a new line at the end of file?

Reply:
updated.